sdk/tests/deqp/framework/opengl/simplereference/sglrReferenceContext.js

```diff
@@ -3636,130 +4135,152 @@ goog.scope(function() {
                 dst = texture.getLevel(level);
                 dst.clear([0.0, 0.0, 0.0, 1.0]);
             }
-        }
-        // else if (target == gl.TEXTURE_CUBE_MAP_NEGATIVE_X ||
-        //          target == gl.TEXTURE_CUBE_MAP_POSITIVE_X ||
-        //          target == gl.TEXTURE_CUBE_MAP_NEGATIVE_Y ||
-        //          target == gl.TEXTURE_CUBE_MAP_POSITIVE_Y ||
-        //          target == gl.TEXTURE_CUBE_MAP_NEGATIVE_Z ||
-        //          target == gl.TEXTURE_CUBE_MAP_POSITIVE_Z)
-        // {
-        //     // Validate size and level.
-        //     RC_IF_ERROR(width != height || width > m_limits.maxTextureCubeSize || depth != 1, gl.INVALID_VALUE, RC_RET_VOID);
-        //     RC_IF_ERROR(level > deLog2Floor32(m_limits.maxTextureCubeSize), gl.INVALID_VALUE, RC_RET_VOID);
+        } else if (target == gl.TEXTURE_CUBE_MAP_NEGATIVE_X ||
+                 target == gl.TEXTURE_CUBE_MAP_POSITIVE_X ||
+                 target == gl.TEXTURE_CUBE_MAP_NEGATIVE_Y ||
+                 target == gl.TEXTURE_CUBE_MAP_POSITIVE_Y ||
+                 target == gl.TEXTURE_CUBE_MAP_NEGATIVE_Z ||
+                 target == gl.TEXTURE_CUBE_MAP_POSITIVE_Z) {
+            // Validate size and level.
+            if (this.condtionalSetError(width != height || width > this.m_limits.maxTextureCubeSize || depth != 1, gl.INVALID_VALUE))
+                return;
+            if (this.condtionalSetError(level > Math.floor(Math.log2(this.m_limits.maxTextureCubeSize)), gl.INVALID_VALUE))
+                return;
 
-        //     TextureCube*    texture = unit.texCubeBinding ? unit.texCubeBinding : &unit.defaultCubeTex;
-        //     tcu::CubeFace face = mapGLCubeFace(target);
+            /** @type {sglrReferenceContext.TextureCube} */
```

Comment:
The right hand annotation is type casting. I suppose it should be enough for closure to figure out the type of the variable.